docs/markdown/series.md

```diff
@@ -36,6 +36,10 @@ Exact size for all series points in pixels or a series object.
 Type: `number|Object`  
 Exact opacity for all series points in pixels or a series object.
 
+### cluster (optional, `BarSeries` only)
```

Comment:
can you write a todo to remove this when we get a bar series documentation file?